arangodb-net-standard/DocumentApi/DocumentApiClient.cs

```diff
@@ -371,11 +388,18 @@ public async Task<DeleteDocumentsResponse<T>> DeleteDocumentsAsync<T>(string col
         /// there.
         /// PATCH/_api/document/{document-handle}
         /// </summary>
+        /// <typeparam name="T">Type of the document to patch.</typeparam>
```

Comment:
I think the wording here could be confusing - "the type of the document to patch" makes me think of the ArangoDB document rather than the patch object.  I would describe something like "Type of the patch object used to partially update a document."

Reply:
i.e. "the document to patch" is the subject of the patch operation, not the same as "the patch" itself.

Reply:
Thanks Ross, I agree. I will update.